src/java/org/apache/cassandra/cql3/statements/schema/CopyTableStatement.java

```diff
@@ -83,7 +86,8 @@ public CopyTableStatement(String sourceKeyspace,
         this.sourceTableName = sourceTableName;
         this.targetTableName = targetTableName;
         this.ifNotExists = ifNotExists;
-        this.createLikeOption = createLikeOption;
+        this.createLikeOptions = createLikeOptions != null ?
+            EnumSet.copyOf(createLikeOptions) : EnumSet.noneOf(CreateLikeOption.class);
```

Comment:
nit: codestyle for ternary operator
```suggestion
        this.createLikeOptions = createLikeOptions != null
                                 ? EnumSet.copyOf(createLikeOptions)
                                 : EnumSet.noneOf(CreateLikeOption.class);
```